nnstreamer/tensor_trainer/tensor_trainer_nntrainer.hh

```diff
@@ -141,17 +142,13 @@ public:
    * @brief Destroy the InputTensorsInfo object
    */
   ~InputTensorsInfo();
-
-  bool is_data_wait_locked;
-  bool is_data_full_locked;
   unsigned int queue_size;
   unsigned int queue_front;
   unsigned int queue_rear;
-  unsigned int queue_count; /**< The number of data in queue */
-  unsigned int push_count;  /**< The number of samples pushed to queue by
-                               NNStreamer(tensor_trainer) */
-  unsigned int pop_count;   /**< The number of pop from the queue for pushing
-                               samples to nntrainer */
+  unsigned int push_count; /**< The number of samples pushed to queue by
+                              NNStreamer(tensor_trainer) */
+  unsigned int pop_count;  /**< The number of pop from the queue for pushing
+                              samples to nntrainer */
```

Comment:
Leave "API" methods only for public. Make all the others private or protected. For C API function accessing private info, add friend. Don't leave private properties public, future developers will abuse them.

Reply:
Thank you for your detailed review, I will update PR ASAP.
Thank you.